django_ninja_crudl/service.py

```
@@ -1,362 +1,362 @@
+#             action="create",
+#             request=request,
+#             schema=self.config.create_schema,
+#             path_args=path_args,
+#             payload=payload,
+#             model_class=self.config.model,
+#         )
+#         if not self.has_permission(request_details):
+#             return self.get_403_error(request)
+#         self.pre_create(request_details)
+#
+#         obj_fields_to_set: list[tuple[str, Any]] = []  # pyright: ignore[reportExplicitAny]
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)